server/apps/main/tests/test_helpers.py

```diff
@@ -176,4 +180,23 @@ def test_delete_pe(self):
         self.assertEqual(1,len(PublicExperience.objects.filter(experience_id='TEST_ID')))
         # Try deleting PE from third party, should not delete!
         delete_PE("TEST_ID",self.moderator_user)
-        self.assertEqual(0,len(PublicExperience.objects.filter(experience_id='TEST_ID')))
\ No newline at end of file
+        self.assertEqual(0,len(PublicExperience.objects.filter(experience_id='TEST_ID')))
+
+
+    def test_model_to_form(self):
+        """
+        Test that turning a PE object into a moderation form works
+        """
+        self.public_experience = PublicExperience.objects.create(
+            experience_text = 'EXP_TEXT', 
+            difference_text = 'DIFF_TEXT', 
+            title_text = 'TITLE',
+            experience_id = 'TEST_ID',
+            open_humans_member = self.moderator_user,
+            abuse = True, 
+            violence = True,
+        )
+        self.public_experience.save()
+        self.form = model_to_form(self.public_experience)
```

Comment:
I found another legacy bit that could be removed I think: The `disable_moderator` parameter and logic in the `model_to_form` for the `ModerateExperienceForm` seem to be superfluous now, since I modified the form to not include the actual experience text etc?

Reply:
That sounds sensible - but let me wait until this evening to get it completely wrong again :-)